frontend/src/scenes/sessions/sessionsTableLogic.ts

```diff
@@ -73,37 +94,41 @@ export const sessionsTableLogic = kea<sessionsTableLogicType<Moment, SessionType
             }
             actions.appendNewSessions(response.result)
         },
-        dateChanged: () => {
-            actions.loadSessions(true)
+        setFilters: () => {
             actions.setNextOffset(null)
+            actions.loadSessions(true)
         },
         previousDay: () => {
-            actions.dateChanged(moment(values.selectedDate).add(-1, 'day'))
+            actions.setFilters(values.filters, moment(values.selectedDate).add(-1, 'day'))
         },
         nextDay: () => {
-            actions.dateChanged(moment(values.selectedDate).add(1, 'day'))
+            actions.setFilters(values.filters, moment(values.selectedDate).add(1, 'day'))
         },
     }),
     actionToUrl: ({ values }) => ({
-        dateChanged: () => {
-            const { selectedDateURLparam } = values
-            const today = moment().startOf('day').format('YYYY-MM-DD')
-            if (window && !window.location.href.includes('person')) {
-                return [`/sessions`, selectedDateURLparam === today ? {} : { date: selectedDateURLparam }]
-            }
+        setFilters: () => {
+            return buildURL(values.selectedDateURLparam, values.filters)
         },
     }),
     urlToAction: ({ actions, values }) => ({
-        '/sessions': (_: any, { date }: { date: string }) => {
+        '/sessions': (_: any, { date, properties }: { date: string; properties: Record<string, any> }) => {
             const newDate = date ? moment(date).startOf('day') : moment().startOf('day')
-            if (!values.selectedDate || values.selectedDate.format('YYYY-MM-DD') !== newDate.format('YYYY-MM-DD')) {
-                actions.dateChanged(newDate)
+            if (
+                !values.selectedDate ||
+                values.selectedDate.format('YYYY-MM-DD') !== newDate.format('YYYY-MM-DD') ||
+                JSON.stringify(properties || {}) !== JSON.stringify(values.filters)
```

Comment:
Q: JSON.stringfy(x) is dependent  on key order, this means:

```
JSON.stringify({ a: 1, b: 2 }) !== JSON.stringify({ b: 2, a: 1 })
```

Is that relevant in this case?